pages/_app.js

```diff
@@ -1,9 +1,36 @@
 import "../styles/globals.css";
 import Layout from "../components/Layout";
+import useLocalStorageState from "../components/Hooks/useLocalStorageState";
+
 function MyApp({ Component, pageProps }) {
+  // Custom hooks setting user user details - here in the _app.js so it can be accessed acrosss all componenets!
+  const [learnerName, setLearnerName] = useLocalStorageState(
+    "learner-name",
+    null
+  ) || [null, null];
+  const [font, setFont] = useLocalStorageState("font", null) || [null, null];
```

Comment:
I really like the use of the local storage use state taking an initial statement to null, once a font is selected then there will be a selected for a specific learner ✨